packages/flutter/lib/src/material/dialog.dart

```diff
@@ -313,7 +314,10 @@ class AlertDialog extends StatelessWidget {
   /// The (optional) set of actions that are displayed at the bottom of the
   /// dialog.
   ///
-  /// Typically this is a list of [FlatButton] widgets.
+  /// Typically this is a list of [FlatButton] widgets. It is recommended to
+  /// set the [textAlign] to [TextAlign.end] for the [Text] within the
+  /// [FlatButton], so that longer actions that result in multiline text align
```

Comment:
so that buttons whose labels wrap to an extra line align with the overall [ButtonBar]'s alignment within the dialog.

Reply:
Done